lmnet/lmnet/utils/horovod.py

```diff
@@ -61,6 +61,8 @@ def _get_pname(pid):
 
 # return True if the parent process is mpirun or horovodrun
 def is_enabled():
+    if os.getenv("USE_HOROVOD"):
```

Comment:
IMHO: How about `ENABLE_HOROVOD`?

Reply:
I named this environment variable after `use_horovod` variable in python, but `ENABLE_HOROVOD` also seems fine.

Reply:
Oh, OK, there is a variable named `use_horovod` so it is fine `USE_HOROVOD`. Thanks.